docs/HowToGuides.md

```diff
@@ -1,5 +1,63 @@
 # How To ...
 
+### Create a multi-step PR
+
+Big PRs (i.e. PRs touching more than 100 lines) are hard to review. 
+Using the _multi-step PR_ format explained below allows the review to be done incrementally.
+
+1. Dev creates the PR branch consisting of a series of commits corresponding to a sequence of logical steps, 
+   each step advancing the code base a step towards the end goal of the PR. The build should pass at each step.
+   <details>
+   <summary>Instructions</summary>
+   
+   * There Ideal approach for creating a multi-step commit sequence: 
```

Comment:
Also, this whole "ideal approach" point seems kind of redundant, given that it never happens in practice (I explain more below)